src/components/RulesDialog.vue

```diff
@@ -1,239 +1,205 @@
-            but this takes your turn, unlike countering.
-          </p>
+        <h4>Can I counter a Two with a Two?</h4>
+        <p>
+          Heck yeah! Playing a Two to counter is a One-Off, which can be countered with a two. Stacked
+          counters make for exciting plays.
+        </p>
 
-          <h4>Do Queens protect against countering with a two?</h4>
-          <p>
-            Yes! Queens protect against targeting, when a card applies a unique effect to one specific card.
-            This means your other cards (not the Queen, herself) are protected against Twos (either effect),
-            Nines, and Jacks.
-          </p>
+        <h4>Can I win by playing a King?</h4>
+        <p>
+          Yes. If you meet the required number of points, you win immediately. This means if you play a King
+          when you already have enough points to meet the new limit, you win on that turn.
+        </p>
 
-          <h4>Can I counter a Two with a Two?</h4>
-          <p>
-            Heck yeah! Playing a Two to counter is a One-Off, which can be countered with a two. Stacked
-            counters make for exciting plays.
-          </p>
+        <h4>If I play an Ace or a Six, are my cards destroyed as well?</h4>
+        <p>
+          Yes. Aces and Sixes destroy all Point Cards and all Face Cards plus Glasses Eights, respectively.
+          That includes any that you have out. Try to avoid destroying many of your own cards!
+        </p>
 
-          <h4>Can I win by playing a King?</h4>
-          <p>
-            Yes. If you meet the required number of points, you win immediately. This means if you play a King
-            when you already have enough points to meet the new limit, you win on that turn.
-          </p>
-
-          <h4>If I play an Ace or a Six, are my cards destroyed as well?</h4>
-          <p>
-            Yes. Aces and Sixes destroy all Point Cards and all Face Cards plus Glasses Eights, respectively.
-            That includes any that you have out. Try to avoid destroying many of your own cards!
-          </p>
-
-          <h4>Where can I play Cuttle?</h4>
-          <p>
-            You can play Cuttle anywhere you have a deck of cards and a friend! If you want to play online,
-            there is only one place:
-            <a href="https://www.cuttle.cards" target="_blank">www.cuttle.cards</a>
-          </p>
-        </section>
-      </v-card-text>
-      <v-card-actions class="d-flex justify-end">
-        <v-btn variant="outlined" color="primary" @click="close">Got It!</v-btn>
-      </v-card-actions>
-    </v-card>
-  </v-dialog>
+        <h4>Where can I play Cuttle?</h4>
+        <p>
+          You can play Cuttle anywhere you have a deck of cards and a friend! If you want to play online,
+          there is only one place:
+          <a href="https://www.cuttle.cards" target="_blank">www.cuttle.cards</a>
```

Comment:
Can we style this link a lighter blue? It's hard to see
<img width="1420" alt="image" src="https://github.com/cuttle-cards/cuttle/assets/6520704/b14b4ee1-5818-46b5-8059-07863164d79f">

Reply:
It also might be nice to set this dialog to `scrollable` so that you can see the actions at the bottom all the time ([docs](https://vuetifyjs.com/en/api/v-dialog/#props-scrollable))

Reply:
Nice, scrollable sounds good.... As for the link, I feel like that section is a little redundant, at that point you're already in the app playing a game and checking the rules.

Reply:
Yeah fair point we can remove it